tmt/utils/__init__.py

```diff
@@ -2965,14 +2949,8 @@ def container_field(
             continue
 
         metadata = field.metadata.get('tmt', FieldMetadata())
-        if inspect.isclass(container):
-            return field.name, key_to_option(field.name), field, metadata
-
-        return (
-            field.name,
-            key_to_option(field.name),
-            container.__dict__[field.name],
-            field, metadata)
+        fname = container.__dict__[field.name] if not inspect.isclass(container) else None
```

Comment:
Is `fname` a fitting name? It looks more like a value of the field.

Reply:
I'm super bad at naming things. What would you suggest?

Reply:
`value`, or maybe not even assign it to a dedicated variable & return it right away:

```python
return (
    field.name,
    key_to_option(field.name),
    container.__dict__[field.name] if not inspect.isclass(container) else None,
    field,
    metadata)

Reply:
Good point, thanks, edited in https://github.com/teemtee/tmt/pull/3142/commits/d0acfc110faca3a70b0d8d8cc1987fd2aa1fd6dc